packages/ometrics/ometrics.0.1.2/opam

```diff
@@ -17,7 +17,7 @@ depends: [
   "csexp" {>= "1.5.1"}
   "result" {>= "1.5"}
   "cmdliner" {<= "1.0.4"}
-  "digestif"
+  "digestif" {>= "0.7"}
```

Comment:
@kit-ty-kate thanks for that. How did you notice that? Do you have a linter or something? So in the future you don't need to do this anymore.

Reply:
We use a special mode in opam in our CI that installs the oldest versions of each dependencies instead of the latest ones:
```
opam install --solver=builtin-0install --criteria="+removed,+count[version-lag,solution]" ometrics.0.1.2
```